src/config/middleware/experimental-metrics-csv-middleware.ts

```diff
@@ -95,7 +99,9 @@ class ExperimentalMetricsCsvMiddlewareRequestHandler
   private recordMetrics(): void {
     const endTime = new Date().getTime();
     const metrics: RequestMetrics = {
-      numActiveRequests: ExperimentalMetricsCsvMiddleware.numActiveRequests,
+      numActiveRequestsAtStart: this.numActiveRequestsAtStart,
+      numActiveRequestsAtFinish:
+        ExperimentalMetricsCsvMiddleware.numActiveRequests,
```

Comment:
Are we including the current request as still active at this point?

Reply:
i would say yes but I don't feel strongly, i can change it if you have a different pref

Reply:
Can just document that `numActiveRequestsAtFinish` includes it the current request

Reply:
https://github.com/momentohq/client-sdk-nodejs/pull/342/commits/02da4b3b9c0553fdf3b957d636ffd401dea32e61